docker-bake.hcl

```diff
@@ -18,6 +18,10 @@ variable "MODULES_COMMIT" {
   default = ""
 }
 
+variable "LIBFAUP_COMMIT" {
+  default = "3a26d0a"
```

Comment:
Like with all other vars, I would init to null here, and set the value in the template.env (and thus .env when building) file.